Kudu.Core/Scripts/kudusync

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
-"$0.NET.exe" "$@"
+mono "$0.NET.exe" "$@"
```

Comment:
We need to make sure that this file is never used on Windows, or it will be broken.